workers/tests/integrationtests/api_response_schema.py

```diff
@@ -49,19 +31,75 @@ def get_response_schema() -> Schema:
     )
 
 
-def get_feature_schema(iterator: Optional[str] = None):
+def get_result_schema() -> Schema:
+    """Response schema for all endpoints except of 'indicator' and 'report' endpoints"""
```

Comment:
I'm a bit confused here. Which other endpoints are this?

Reply:
PR #106 added API-endpoints to list the names of indicators, reports, layers, datasets and feature id fields. Those will be a list at the response key `result`. Those will be available in production once we release version 0.6.0.

Reply:
And there is also the `regions` endpoint to retrive all regions of dataset regions as GeoJSON.